test/integration/features/user_adds_template_to_project.spec.js

```diff
@@ -42,24 +38,42 @@ describe('User adds a template to a project', function() {
             // verify we have the 2 deployments in the template
             var deploymentsPage = new DeploymentsPage(project);
             deploymentsPage.visit();
-            expect(element(by.cssContainingText('td', 'mongodb')).isPresent()).toBe(true);
-            expect(element(by.cssContainingText('td', 'nodejs-mongodb-example')).isPresent()).toBe(true);
+            expect(element(by.cssContainingText('td', 'mongodb')).isPresent()).toBe(true); // TODO: use fixture
```

Comment:
TODO's should be obvious what is needed for someone coming else looking at this later. Also fixture isn't doesn't seem like the right word here, shouldn't it be a helper.